src/OpenLoco/Paint/PaintTree.cpp

```diff
@@ -0,0 +1,103 @@
+#include "PaintTree.h"
+#include "../Config.h"
+#include "../Graphics/Colour.h"
+#include "../Map/Tile.h"
+#include "../Objects/ObjectManager.h"
+#include "../Objects/TreeObject.h"
+#include "../Ui.h"
+#include "Paint.h"
+
+using namespace OpenLoco::Interop;
+using namespace OpenLoco::Ui::ViewportInteraction;
+
+namespace OpenLoco::Paint
+{
+    constexpr std::array<uint8_t, 6> _50076A = { 3, 0, 1, 2, 1, 4 };
+    constexpr std::array<bool, 6> _500770 = { true, true, false, false, true, true };
```

Comment:
These are used by a `season` variable. Any idea why there are six elements?

Also, the values from `_50076A` are used to index into `_500770`. Any idea why are there six elements? The last one is unreachable, isn't it?

Reply:
yeah thats  a good point. The last one is unreachable. As to what it represents, no idea will have to implement more to find out i think.